qemu/tests/cfg/block_stream.cfg

```diff
@@ -94,4 +94,4 @@
                     cancel_timeout_image1 = 3
                     max_speed_image1 = 10M
                     when_streaming = "pause_job set_speed resume_job cancel"
-                    after_finished = "start reboot verify_alive"
```

Comment:
it looks after_finished should be call after wait_for_finished. if not, I think it a bug for blk_stream module.

Thanks,
Xu